test/e2e/myReviews.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { test, expect, Page } from "@playwright/test";
+import { BookPage } from "../pages/BookPage";
+import { MyReviewPage } from "../pages/MyReviewsPage";
+import { LoginPage } from "../pages/LoginPage";
+import { VALID_EMAIL, VALID_PASSWORD } from "../helper-config";
+import { BookDetailPage } from "../pages/BookDetailPage";
+
+let reviewPage: MyReviewPage;
+
+test.beforeEach(async({page}: {page: Page}) => {
+    reviewPage = new MyReviewPage({page});
+    await page.goto(BookPage.MY_BOOKS_URL);
+});
+
+async function performAddReviews(page) {
+    await expect(reviewPage.bookImageLocator).toBeVisible({ timeout: 10000 });
+    const bookDetailPage = await reviewPage.addReviews();
+    await page.goto(BookDetailPage.BOOK_DETAIL_PAGE_URL);
+    await bookDetailPage.reviewIconLink.click();
+    await expect(reviewPage.bookReviewPopupLocator).toBeVisible();
+    await reviewPage.reviewTitleLocator.click();
+    await reviewPage.reviewFeatureLocator.click();
+    await reviewPage.reviewPopupCloseIconLocator.click();
+}
+
+async function performUpdateReviews(page) {
+    await expect(reviewPage.updateReviewsLinkLocator).toBeVisible({ timeout: 10000 });
+    await reviewPage.updateReviews();
+    await expect(reviewPage.bookReviewPopupLocator).toBeVisible();
+    await expect(reviewPage.reviewsContainerLocator).toBeVisible();
+    await reviewPage.selectedReviewCloseIconLocator.click();
+    let selectedReviewsText = await reviewPage.selectedReviewsLocator.allInnerTexts();
+    let cleanedSelectedReviews = selectedReviewsText.map((text)=> text.replace(/×$/, '').trim()); 
+    await reviewPage.reviewPopupCloseIconLocator.click();
```

Comment:
same here, no clicking should be needed in the specs file.
what do you want to do on the page? => create a function in the page object to do that, then call that function from here